src/video-recorder/index.js

```diff
@@ -78,40 +71,22 @@ export default class VideoRecorder {
         this.warningLog.addWarning(WARNING_MESSAGES.problematicPathPatternPlaceholderForVideoRecording, problematicPlaceholderListStr, suffix, suffix, verb);
     }
 
-    _getTargetVideoPath (testRunInfo) {
-        const { test, index, testRun } = testRunInfo;
+    _getTargetVideoPath (testRunRecorder) {
```

Comment:
`TestRunRecorder` is a bad name because it looks like part of the recorder functionality from `TestCafe Studio`.
What about `TestRunVideoRecorder` name?

Reply:
probably, yes